ext/RMagick/rmimage.c

```diff
@@ -2529,10 +2529,7 @@ Image_color_histogram(VALUE self)
     if (image->storage_class != DirectClass)
     {
         dc_copy = rm_clone_image(image);
-        (void) SyncImage(dc_copy);
-        magick_free(dc_copy->colormap);
-        dc_copy->colormap = NULL;
-        dc_copy->storage_class = DirectClass;
+        (void) SetImageStorageClass(dc_copy, DirectClass);
```

Comment:
Can you add test code for this branch condition?

https://github.com/rmagick/rmagick/blob/c00a81d377b31e0c2daf6bf80450663afd5e2dd6/test/Image1.rb#L432-L437

Reply:
Added a unit test to make sure we also hit that branch.